Core/Frameworks/Baikal/Model/Config.php

```diff
@@ -39,7 +39,14 @@ function __construct($sConfigFileSection) {
 
         try {
             $config = Yaml::parseFile(PROJECT_PATH_CONFIG . "baikal.yaml");
-            $aConfig = $config[$sConfigFileSection];
+            if (isset($config[$sConfigFileSection])) {
+                $aConfig = $config[$sConfigFileSection];
+            } else {
+                error_log('Section ' . $sConfigFileSection
+                        . ' not found in config file. Using default values.');
+                $aConfig = array();
```

Comment:
```suggestion
                $aConfig = [];
```

Reply:
Man, I really need to install the php-cs-fixer plugin for my IDE :D